compiler/backend/src/org/jetbrains/kotlin/codegen/inline/LambdaInfo.kt

```diff
@@ -130,84 +100,52 @@ abstract class DefaultLambda(
     final override lateinit var invokeMethod: Method
         private set
 
-    private lateinit var invokeMethodDescriptor: FunctionDescriptor
-
-    override val invokeMethodParameters: List<KotlinType?>
-        get() = invokeMethodDescriptor.valueParameters.map { it.returnType } // should be FunctionN, so no extension receiver
```

Comment:
Is old assumption is broken?

Reply:
I mean ` // should be FunctionN, so no extension receiver` ?

Reply:
It's fine, it's just safer this way I think